python/nav/mibs/comet.py

```diff
@@ -63,6 +63,8 @@ def get_channels(self):
 
             name = yield self.get_next(o_name)
             unit = yield self.get_next(o_unit)
+            if not name and not unit:
```

Comment:
Without this the old comet code would incorrectly add broken sensors and preventing the new module from running on the comet ms probes. It should do nothing for the existing supported probes